pkg/apis/core/types.go

```diff
@@ -6203,7 +6203,7 @@ type SecurityContext struct {
 	// +optional
 	AllowPrivilegeEscalation *bool
 	// ProcMount denotes the type of proc mount to use for the containers.
-	// The default is DefaultProcMount which uses the container runtime defaults for
+	// The default value is Default which uses the container runtime defaults for
```

Comment:
good catch, however, these are the internal types, the ones we need to update for the documentation are here

https://github.com/kubernetes/kubernetes/blob/eb66365bc4a84eca9b7f4ca6436535bc78d46648/staging/src/k8s.io/api/core/v1/types.go#L7454-L7460

so once you update both places, do `make update` locally to regenerate the documentation and push all together

Reply:
thanks for the pointers, I did as you said.